README.md

```diff
@@ -29,4 +29,3 @@ For participating in this contest, register [here](https://bit.ly/3c3lNQY) and g
 -   To ask for help, feel free to open a new issue in this repository.
 
 *Happy Contributing!*
-
```

Comment:
You seem to have deleted that last line which is not required. Please add it back so there are no visible changes in README.md

Reply:
can u please review the changes now. I have tried to make README.md file like original one.If u could please check and tell me.

Reply:
@ishikabansal04, you are left with just one more commit and that is adding a blank line at line no. 32. As you can see above, the pink highlighted line is the one that you removed. Git keeps a track of the minutest changes and that includes removal of even an empty line. Add that line back to the README.md file and your PR is then good to merge.

Reply:
I have added line no 32 again.
@soumyaa1804

Reply:
@ishikabansal04, I just saw the last line number of the master is 33. You also need to add it so that it shows only one file change.